frontend/src/metabase/entities/dashboards.js

```diff
@@ -119,18 +156,13 @@ const Dashboards = createEntity({
   },
 
   reducer: (state = {}, { type, payload, error }) => {
-    if (type === FAVORITE_ACTION && !error) {
-      return assocIn(state, [payload, "favorite"], true);
-    } else if (type === UNFAVORITE_ACTION && !error) {
-      return assocIn(state, [payload, "favorite"], false);
-    } else if (type === COPY_ACTION && !error && state[""]) {
+    if (type === COPY_ACTION && !error && state[""]) {
```

Comment:
I know it's written by someone else, but what's the purpose of `state[""]`?
<img width="167" alt="image" src="https://github.com/metabase/metabase/assets/125459446/b1eeb564-e036-4512-a480-127c636d4ca7">

Reply:
No idea. But it's potentially not useless:

```
x = { "": "hello" };
x[""] // "hello"
```